### PR TITLE
OCPBUGS-29402: Adding safe sysctls

### PR DESCRIPTION
**Pull request by rohennes**
OCPBUGS-29402: Adding missing safe sysctl parameters.

Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OCPBUGS-29402

Link to docs preview:
https://78983--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/containers/nodes-containers-sysctls.html#safe_and_unsafe_sysctls_nodes-containers-using

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by rohennes** [comment]
Approved by Ben Niven-Jenkins in Slack.

File: modules/nodes-safe-sysctls-list.adoc [hunk 1]
```diff
@@ -49,6 +49,22 @@ a| This restricts `ICMP_PROTO` datagram sockets to users in the group range. The
 
 | `net.ipv4.ip_unprivileged_port_start`
 | This defines the first unprivileged port in the network namespace. To disable all privileged ports, set this to `0`. Privileged ports must not overlap with the `ip_local_port_range`. For more information, see link:https://docs.kernel.org/networking/ip-sysctl.html?highlight=ip_unprivileged_port_start#ip-variables#ip-variables[ip_unprivileged_port_start].
+
+|`net.ipv4.ip_local_reserved_ports`
+| Specify a range of local ports to reserve for applications or services. Use comma-separated values.
```
Comment (agantony) [inline modules/nodes-safe-sysctls-list.adoc:54]:
```suggestion
| Specify a range of comma-separated local ports that you want to reserve for applications or services.
```

File: modules/nodes-safe-sysctls-list.adoc [hunk 2]
```diff
@@ -49,6 +49,22 @@ a| This restricts `ICMP_PROTO` datagram sockets to users in the group range. The
...
+
+| `net.ipv4.tcp_keepalive_time`
+| Specify the interval in seconds before the first `keepalive` probe should be sent after a connection has become idle.
+
+| `net.ipv4.tcp_fin_timeout`
+| Specify the time in seconds that a connection stays in the `FIN-WAIT-2` state before it is aborted.
```
Comment (agantony) [inline modules/nodes-safe-sysctls-list.adoc:60]:
```suggestion
| Specify the time in seconds that a connection remains in the `FIN-WAIT-2` state before it is aborted.
```

File: modules/nodes-safe-sysctls-list.adoc [hunk 3]
```diff
@@ -49,6 +49,22 @@ a| This restricts `ICMP_PROTO` datagram sockets to users in the group range. The
...
+
+| `net.ipv4.tcp_keepalive_intvl`
+| Specify the interval in seconds between `keepalive` probes. This value is multiplied by the `tcp_keepalive_probes` value to determine the total time designated before deciding that the connection is broken.
```
Comment (agantony) [inline modules/nodes-safe-sysctls-list.adoc:63]:
```suggestion
| Specify the interval in seconds between the `keepalive` probes. This value is multiplied by the `tcp_keepalive_probes` value to determine the total time required before it is decided that the connection is broken.
```

File: modules/nodes-safe-sysctls-list.adoc [hunk 4]
```diff
@@ -49,6 +49,22 @@ a| This restricts `ICMP_PROTO` datagram sockets to users in the group range. The
...
+
+| `net.ipv4.tcp_keepalive_probes`
+| Specify how many `keepalive` probes to send until determining that the connection is broken.
```
Comment (agantony) [inline modules/nodes-safe-sysctls-list.adoc:66]:
```suggestion
| Specify how many `keepalive` probes to send until you determine that the connection is broken.
```

**Review by agantony — COMMENTED** [review]
Added a few suggestions for your consideration; overall lgtm!

File: modules/nodes-safe-sysctls-list.adoc (hunk 4 above)
Comment (rohennes) [reply modules/nodes-safe-sysctls-list.adoc:66]: Can't avoid passive tense here. Changed to "until it is determined" for parallelism with other rows at least.

Comment (mburke5678) [reply modules/nodes-safe-sysctls-list.adoc]: @rohennes 
For future thought: The term _determined_ seems a bit anthropomorphic. I wonder if _detects_ would be better? Not worth blocking the merge.

**Pull request merged**